buildscripts/AssemblyMetadata.cs

```diff
@@ -0,0 +1,7 @@
+internal static class AssemblyMetadata
+{
+	public const string MajorVersion = "3";
+	public const string MinorVersion = "3";
+	public const string BuildVersion = "4";
+	public const string BuildNumber = "0";
+}
```

Comment:
This would be the new place to set the version number. `Get-NuGetVars.ps1` has been changed to parse this instead of `Settings.proj` (which could maybe lose the version numbers?)